File: 2do Semestre/Taller/Modulo Objetos/Práctica 3/Cliente.java
package tema3;

public class Cliente {
    private String nombre;
    private String DNI;
    private int edad = 0;
    
    public Cliente(String nombre, String DNI, int edad) {
        this.nombre = nombre;
        this.DNI = DNI;
        this.edad = edad;
    }
    
    //Getters
    public String getNombre() {
        return this.nombre;
    }
    
    public String getDNI() {
        return this.DNI;
    }
    
    public int getEdad() {
        return this.edad;
    }
    
    //Setters
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public void setDNI(String DNI) {
        this.DNI = DNI;
    }

    public void setEdad(int edad) {
        this.edad = edad;
    }

    @Override
    public String toString() {
        return "Cliente{" + "nombre=" + nombre + ", DNI=" + DNI + ", edad=" + edad + '}';
    }
}

File: 3er Semestre/Seminario .NET/Práctica 10/Ejercicio 4/Cliente.cs
namespace Tienda;

public class Cliente
{
    public int Id { get; set; }
    public string? DNI { get; set; }
    public string? ApellidoYNombre { get; set; }
    public string? Direccion { get; set; }
    public string? Mail { get; set; }
    public string? Telefono { get; set; }

    public override string ToString()
    {
        return "{ Id = " + Id + ", DNI = " + DNI +", Nombre = " + ApellidoYNombre +", Direccion = " + Direccion + ", Tel = " + Telefono + " }";
    }
}
